thunder/dynamo/utils.py

```diff
@@ -742,6 +743,17 @@ def reproducer(
     readable = _readable(gm, "DynamoModule", print_output=False)
     has_cuda_args = any(hasattr(arg, "device") and arg.device.type == "cuda" for arg in args)
     thunder_options_str = thunder_options_to_str(thunder_options)
+
+    # split reason
+    if subgraph_info.split_reasons:
```

Comment:
This is great! For completeness, I suggest always printing something. For example, in the case of a single subgraph sent to Thunder, maybe this could print

```
Split Information: 
The original graph is not split, and is entirely run by thunder.
```

That way there can always be a split information section, so people looking for it can find it and verify what happened.

Nit: Maybe just say `thunder` instead of `thunder.jit` in the current text?

Reply:
Yeah, thanks for the tip, it's better to always print the split information, I've added it.
Hi @tfogal , would you like to also take a look?